deployments/container/Makefile

```diff
@@ -29,13 +29,16 @@ endif
 
 IMAGE_VERSION := $(VERSION)
 
+IMAGE_EXTRA_TAG_NAMES ?=
 IMAGE_TAG ?= $(IMAGE_VERSION)-$(DIST)
 IMAGE = $(IMAGE_NAME):$(IMAGE_TAG)
+IMAGE_EXTRA_TAGS := $(foreach tag,$(IMAGE_EXTRA_TAG_NAMES),$(IMAGE_NAME):$(tag))
```

Comment:
We lose the dist-specific tags? This means that for cases where we do produce two distributions they specified tags will be overridden.